framework/external/easylogging/easylogging++.h

```diff
@@ -2897,6 +2897,9 @@ return *this;\
   inline MessageBuilder& operator<<(const std::string& msg) {
     return operator<<(msg.c_str());
   }
+  inline MessageBuilder& operator<<(uint8_t c) {
+    return operator<<(+c);
+  }
```

Comment:
We should add the same operator for signed char (int8_t).

Reply:
That's a bit more risky, because it would make `LOG(DEBUG) << foo << '\n';` print "10" instead of a newline... And it's not so easy to find occurences where a char is really meant as a character.